classes/fields/time.php

```diff
@@ -218,7 +234,7 @@ public function format_time( $options, $js = false ) {
 				$format = get_option( 'time_format' );
 
 				if ( $js ) {
-					$format = $this->convert_format( $format, array( 'source' => 'php' ) );
+					$format = $this->convert_format( $format, array( 'source' => 'php', 'type' => 'time' ) );
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: fb597aa965bf20decdbac32a176b4e78</sub>